test/data/auth.contexts.yaml

```diff
@@ -2,3 +2,4 @@
 -
   context: this is a context,
   displayName: this is a display name
+  autoDeployKeyGeneration: true
```

Comment:
Minor comment, but just wondering why there's `autoDeployKeyGeneration` vs `autoKeysGeneration`?
Would it be better to make them the same?

Reply:
So actually these keys actually carry flags from 2 different sources. `autoDeployKeyGeneration` from the API config and `autoKeysGeneration` from the UI after the user has confirmed. So I kept them different in order to differentiate them. But I guess the UI flag can be renamed to something better.

Reply:
Yeah, I had similar confusion as well. https://github.com/screwdriver-cd/ui/pull/568#discussion_r465889020